config/locales/client.en.yml

```diff
@@ -4056,13 +4063,15 @@ en:
             unread_with_count: "Unread (%{count})"
             archive: "Archive"
         tags:
-          no_tracked_tags: "You are not tracking any tags."
+          no_tags: "You have not added any tags. <a href='%{url}'>Click here to get started.</a>"
           header_link_title: "all tags"
           header_link_text: "Tags"
+          header_action_title: "edit your sidebar tags"
         categories:
-          no_tracked_categories: "You are not tracking any categories."
+          no_categories: "You have not added any categories. <a href='%{url}'>Click here to get started.</a>"
```

Comment:
I always feel a teeny tiny uneasy when I see HTML in translations. I wonder if it's possible to split off `Click here to get started.` into a separate translation and then in the handlebars template we linkify it like this?

```hbs
<a href={{link}}>{{i18n "..."}}</a>
```